<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>HttpProxyLoginWindow</class>
 <widget class="QDialog" name="HttpProxyLoginWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>@product@ - Local HTTP Proxy</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="httpProxyLabel" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>10</red>
         <green>36</green>
         <blue>106</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>233</red>
         <green>231</green>
         <blue>227</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>10</red>
         <green>36</green>
         <blue>106</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string>&lt;span style="font-size:18pt;">HTTP Proxy Authentication&lt;/span>&lt;br>&lt;br>&lt;span>Your local HTTP proxy uses authentication&lt;br>Please enter your login and password&lt;/span></string>
     </property>
     <property name="indent" >
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="addressLineEdit" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="addressLabel" >
     <property name="text" >
      <string>HTTP Proxy Address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="portLineEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="portLabel" >
     <property name="text" >
      <string>HTTP Proxy Port Number:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="loginLineEdit" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="loginLabel" >
     <property name="text" >
      <string>HTTP Proxy Login:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="passwordLineEdit" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="passwordLabel" >
     <property name="text" >
      <string>HTTP Proxy Password:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>213</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../../../resources/wengophone.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>HttpProxyLoginWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>252</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel" >
     <x>196</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>HttpProxyLoginWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>334</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel" >
     <x>304</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
